.gitignore

```diff
@@ -1,3 +1,5 @@
+/doc/_build
+/vendor
 composer.lock
-vendor
+phpunit.xml
 .php_cs.cache
```

Comment:
Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore